src/modules/love/callbacks.lua

```diff
@@ -134,6 +137,9 @@ function love.createhandlers()
 				love.audio.setPlaybackDevice()
 			end
 		end,
+		sensorupdate = function (sensorType, x, y, z)
```

Comment:
This (and `joysticksensorupdate` and the event strings) should be past-tense `updated` instead of `update` IMO, to match the other input event names.